web-components/navbar/navbar-btn.css

```diff
@@ -0,0 +1,36 @@
+button {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    display: inline-block;
+    padding-right: 0;
+    outline: none;
+    -webkit-tap-highlight-color: rgba(0,0,0,0) !important;
+}
+div {
```

Comment:
This could style other divs, that we don't want, so it should be more specific (should have a class, and use it)